maven/bnd-resolver-maven-plugin/src/main/java/aQute/bnd/maven/resolver/plugin/ResolverMojo.java

```diff
@@ -0,0 +1,84 @@
+package aQute.bnd.maven.resolver.plugin;
```

Comment:
Actually do we expect this mojo to mainly be on projects with packaging type 'pom'?  If so then lifecycle mappings are not needed.